modules/renter/contractor/contracts.go

```diff
@@ -28,6 +28,16 @@ func (c *Contractor) contractEndHeight() types.BlockHeight {
 	return c.currentPeriod + c.allowance.Period
 }
 
+// ContractUtility returns the ContractUtility for a contract with a given id.
+func (c *Contractor) contractUtility(id types.FileContractID) modules.ContractUtility {
+	sc, exists := c.contracts.Acquire(c.resolveID(id))
```

Comment:
It shouldn't be necessary to `Acquire` the contract to view its utility; `View` should work fine.